hack/generate-manifest.sh

```diff
@@ -264,9 +270,9 @@ if $IPSEC; then
     # create a K8s Secret to save the PSK (pre-shared key) for IKE authentication.
     $KUSTOMIZE edit add resource ipsecSecret.yml
     # add a container to the Agent DaemonSet that runs the OVS IPSec and strongSwan daemons.
-    $KUSTOMIZE edit add patch ipsecContainer.yml
+    $KUSTOMIZE edit add patch --path ipsecContainer.yml
```

Comment:
is specifying the `--path` required by the Kustomize update? If not, why make changes to the existing commands?

Reply:
yes, that's required by Kustomize update. One of `--path` or `--patch` must be given in the CLI parameter. see [the validation code](https://github.com/kubernetes-sigs/kustomize/blob/kustomize/v3.8.8/kustomize/internal/commands/edit/add/addpatch.go#L61)